configure

```diff
@@ -5083,6 +5086,20 @@ else
 fi
 
 
+# Variables for tags utilities; see am/tags.am
+if test -z "$CTAGS"; then
+  CTAGS=ctags
+fi
+
+if test -z "$ETAGS"; then
+  ETAGS=etags
+fi
+
+if test -z "$CSCOPE"; then
+  CSCOPE=cscope
+fi
+
+
```

Comment:
This part and line 653-655 were intentionally left out in the past, due to older autotools (mostly sledgehammer999 is using it) won't generate them.

Reply:
>due to older autotools

Should the `build-aux` scripts be regenerated too? -> https://github.com/qbittorrent/qBittorrent/pull/11384#issuecomment-1151742204

Reply:
>Should the build-aux scripts be regenerated too? -> https://github.com/qbittorrent/qBittorrent/pull/11384#issuecomment-1151742204

If you want to... but then you'll have to re-run bootstrap.sh again.

Reply:
> > Should the build-aux scripts be regenerated too? -> [#11384 (comment)](https://github.com/qbittorrent/qBittorrent/pull/11384#issuecomment-1151742204)
> 
> If you want to... but then you'll have to re-run bootstrap.sh again.

I will leave this for someone else if it really needs updating & I'm unsure if it's just those 2 files is all that's needed before bootstrapping....